phys2bids/bids_units.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+LGR = logging.getLogger(__name__)
+
+unit_aliases = {
```

Comment:
We're going to want to re-order this dictionary such that there are fewer clashes. For example, if I pass:

``` python
>>> bidsify_unit('nanonetwon')
```

I get:

```
The given unit prefix nanonewto does not have aliases, passing it as is
'nanonewton'
```

because the `'n'` key is hit before the 'newton'`.

Similarly, if I pass:

```python
>>> bidsify_unit('1/hertz')
```

I get the output:

```
The given unit prefix 1/ does not have aliases, passing it as is
'1/hertz'
```

because `'hertz'` is checked before `'1/hertz'`. 

There aren't a TON of these clashes, but they will be important so we should either make the `bidsify_unit()` function a bit more robust to these cases or re-order the `unit_aliases` dictionary to attempt to avoid them. I _think_ the latter might be the easier thing to do, but maybe that's naive of me 🤷‍♂️

Reply:
Okay, I reordered the dictionary and even more I added a loop to the test function that checks all the basic units work. This could be useful when adding new units in the dict. Do you think is an overkill?